resource_packs/packs.json

```diff
@@ -451,7 +451,7 @@
 				{
 					"id": "variated_moss",
 					"name": "Variated Moss",
-					"description": "Randomly rotates Moss Blocks and Moss Carpets to break ugly grid patterns"
+					"description": "Randomly rotates Moss Blocks and Moss Carpets (and Pale Moss Blocks and Carpets in 1.21.50+) to break up the ugly grid patterns"
```

Comment:
Change the description of the pack so that there are no brackets and don't specify 1.21.50+ as this pack will only work in 1.21.50+, it will throw an error in 1.21.40 and below so change the description accordingly